packages/tools/viewer/src/viewerElement.ts

```diff
@@ -974,20 +996,23 @@ export abstract class ViewerElement<ViewerClass extends Viewer = Viewer> extends
             }, new Array<TemplateResult>());
         }
 
+        if (toolbarControls.length > 0) return html`<div part="tool-bar" class="bar ${this._hasAnimations ? "" : "bar-min"} tool-bar">${toolbarControls}</div>`;
+        else return html``;

```

Comment:
I'm surprised prettier didn't reformat this and that the build didn't complain (everywhere else in Babylon we have linting with prettier verified as part of the build, so I must have missed something in the way the Viewer is integrated into the standard Babylon build, which I will try to get fixed).
```suggestion
        if (toolbarControls.length > 0) {
            return html`
                <div part="tool-bar" class="bar ${this._hasAnimations ? "" : "bar-min"} tool-bar">
                    ${toolbarControls}
                </div>`;
        } else {
            return html``;
        }
```